### ISSUE TITLE
Define certain types as `enum`s

### ISSUE THREAD

**Issue by jgcodes2020** [larksuite/rsmpeg#144]
As an example, `AVMediaType` could be enum-ified as such:
```rs
#[repr(C)]
enum AVMediaType {
    Unknown = rusty_ffmpeg::ffi::AVMediaType_AVMEDIA_TYPE_UNKNOWN,
    Video = rusty_ffmpeg::ffi::AVMediaType_AVMEDIA_TYPE_VIDEO,
    Audio = rusty_ffmpeg::ffi::AVMediaType_AVMEDIA_TYPE_AUDIO,
    Data = rusty_ffmpeg::ffi::AVMediaType_AVMEDIA_TYPE_DATA,
    Subtitle = rusty_ffmpeg::ffi::AVMediaType_AVMEDIA_TYPE_SUBTITLE,
    Attachment = rusty_ffmpeg::ffi::AVMediaType_AVMEDIA_TYPE_ATTACHMENT,
}
```

**Comment by jgcodes2020** [comment]
This will be solved by upstream [CCExtractor/rusty_ffmpeg#109](https://github.com/CCExtractor/rusty_ffmpeg/pull/109)

**Comment by jgcodes2020** [comment]
Closing for now.
